services-directions/src/main/java/com/mapbox/api/directions/v5/DirectionsService.java

```diff
@@ -95,6 +95,7 @@ Call<DirectionsResponse> getCall(
     @Query("depart_at") String departAt,
     @Query("max_height") Double maxHeight,
     @Query("max_width") Double maxWidth,
+    @Query("max_weight") Double maxWeight,
```

Comment:
We're missing a `MapboxDirections` test for this.

Reply:
in progress...

Reply:
added